components/navigation/DesktopNav.tsx

```diff
@@ -52,6 +137,12 @@ const DesktopNav = () => {
       >
         <MessageRouteIcon width={38} height={38} pathName={pathname} />
         Message
+        {notification && (
+          <div
+            className='absolute right-4 top-3 z-50 h-3 w-3 rounded-full border-2 
+              border-green-700 bg-[#54BB89] shadow-lg outline-4 outline-black'
+          ></div>
```

Comment:
Apologies for the closing tag :disappointed: 
I can do that but the only thing that is different between the dots is it's sizing, but I can change it with tailwind on the element itself.

Reply:
So I found what was the issue with absolute I didn't import the `messaging-styles.css` after doing that it works fine. Also because I didn't do that the dot was not appearing in other pages unless you visited conversations page so made myself confused why the code wasn't working anymore silly me :sweat_smile: